index.d.ts

```diff
@@ -188,6 +188,24 @@ export interface ThemeSectionProperties {
 
 export type ThemePropertyType = 'color' | 'font' | 'backgroundSelector' | 'backgroundColor';
 
+export type Orientation = 'landscape' | 'portrait';
+
+export type WidgetType = 'main' | 'banner' | 'ticker';
+
+export interface Layout {
+  orientation: Orientation;
+  widgetType?: WidgetType;
+  zones?: number;
+  position?: number;
+  flip?: boolean;
+}
+
+export interface FeedData {
+  socialItems?: Array<any>;
```

Comment:
If the type of values in these arrays can't be specified on this level I'd still steer away from `any`. 
Would it make sense to make `FeedData` a generic? i.e.
```typescript
interface FeedData <T> {
  socialItems?: T[];
 /* ... */
}
```